site/content/blogs/release-v123.en.md

```diff
@@ -52,7 +52,7 @@ http:
 By default, your private service can only send traffic to the internet.
 If you'd like to a send traffic to your environment, set [`network.vpc.placement: 'private'`](../../docs/manifest/rd-web-service/#network-vpc-placement) in your manifest.
 
-## Support Aurora Serverless v2 in `storage init`
+## Support Aurora Serverless v2 in [`storage init`](../docs/commands/storage-init.en.md)
```

Comment:
i wonder if we should just remove this section down here?

Reply:
Details for the "detailed section" coming this afternoon!

Reply:
chatted offline - adding details in a separate PR :)